views/metabox/cpcss/generate.php

```diff
@@ -38,7 +38,7 @@
 </p>
 <div class="components-panel__row cpcss_generate cpcss_regenerate">
 	<button id="rocket-generate-post-cpss" class="button components-button is-secondary" <?php disabled( $data['disabled'] ); ?>>
-		<span class="spinner"></span>
+		<span style="display: none;" class="spinner"></span>
```

Comment:
why not use the `hidden` class and remove it when needed, instead of an inline style?

Reply:
If I will use the `hidden` class, then I will have to modify also the CSS file. This is because:
`.spinner` has the following rules:
```
display: inline-block;
visibility: hidden;
```
And it will take priority over `hidden` class rules. So I will have to add a new CSS rule which will take priority with display: none;. 

In the HTML files are we allowed to use inline CSS code? I am sure that is preffered not to use any CSS code inside the `template` files, however for this case what is the direction?